src/services/identity/abstract.ts

```diff
@@ -85,7 +85,12 @@ export abstract class AbstractIdentityService implements IIdentityService {
 		throw new Error(`Not supported`);
 	}
 
-	createResource(network: string, payload: ResourcePayload, customer: CustomerEntity): Promise<any> {
+	createResource(
+		network: string,
+		payload: ResourcePayload,
+		customer: CustomerEntity,
+		publicKeyHexs?: string[]
```

Comment:
Adding as an all-inclusive reference to `publicKeyHexs` introduction to DID, DLR operations:

This assumes only keyrefs can be passed, therefore no external key importing is supported, by passing it as signature material information, otherwise known as `signInput`.

The functionality per se is exported only at the key / did import layer, which is sensible since we're running custodian mode for issuers, verifiers.